ckan/templates/ckan-deployment.yaml

```diff
@@ -130,6 +130,16 @@ spec:
         - name: ckan-conf-templates
           mountPath: /etc/ckan-conf/templates
           readOnly: true
+        {{ if .Values.enableGiftless }}
+        - mountPath: /etc/ssh/jwt-rs256.key
+          name: jwt-key
+          readOnly: true
+          subPath: jwt-rs256.key
```

Comment:
is the subPath needed here?

Reply:
Not sure, probably not, but it works as with it. Anyway, will try removing